x-pack/plugins/secops/public/components/timeline/properties/helpers.tsx

```diff
@@ -58,7 +58,7 @@ export const StarIcon = pure<{
   timelineId: string;
   updateIsFavorite: UpdateIsFavorite;
 }>(({ isFavorite, timelineId: id, updateIsFavorite }) => (
-  <div onClick={() => updateIsFavorite({ id, isFavorite: !isFavorite })}>
+  <div role="button" onClick={() => updateIsFavorite({ id, isFavorite: !isFavorite })}>
```

Comment:
note: This would probably be better later on to be a button. The linter complained with this error:

`react-a11y-event-has-role`
https://github.com/joaovieira/tslint-react-a11y
https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/button_role

Simplest fix for now is to add the role of that of a button.